tests/db_test.js

```diff
@@ -21,6 +21,7 @@ tape('testing post request', t => {
   runDbBuild((err, response) => {
     postQueries.getHomePagePosts((err, res) => {
       if (err) {
+      // eslint-disable-next-line no-console
         console.log(`error:  + ${err}`);
```

Comment:
I asked you to please add in a comment saying something like `// TODO: error handling` if you were going to disable a rule, to show it is a temporary thing.

Reply:
I opened an issue that adress those errors #22